src/OrleansRuntime/LogViews/GrainStateWithMetaData.cs

```diff
@@ -14,8 +14,14 @@ namespace Orleans.Runtime.LogViews
     [Serializable]
     public class GrainStateWithMetaDataAndETag<TView> : IGrainState where TView: class, new()
     {
+        /// <summary>
+        /// State and meta data
+        /// </summary>
         public GrainStateWithMetaData<TView> StateAndMetaData { get; set; }
        
+        /// <summary>
+        /// Etag
+        /// </summary>
```

Comment:
Properties should start with "Gets or sets ..." Or just "Gets ..." if it has no setter. Also similarly use a single line.
If this member is inherited, consider using `<inheritdoc />`